net-mgmt/pfSense-pkg-arpwatch/files/usr/local/pkg/arpwatch.xml

```diff
@@ -117,6 +117,20 @@
 			<type>checkbox</type>
 			<description>Reset the database of collected mac/ip addresses before starting Arpwatch.</description>
 		</field>
+		<field>
+			<fielddescr>Suppress MAC</fielddescr>
+			<fieldname>none</fieldname>
+			<type>rowhelper</type>
+			<rowhelper>
+				<rowhelperfield>
```

Comment:
It would be useful to only suppress certain types of notifications per-MAC.  For example, I might want to suppress "flip flop" but keep "new activity", "new station", and "changed ethernet address".

Reply:
I've implemented and tested my suggested improvement and submitted a PR to your fork.

Reply:
Thank you!
I did some code cleanup and switched mac&type fields (type is more optional)

Reply:
Great!  Looks good. I'm not that experienced with php so I did what I knew would work.  I thought of doing multi-select on the options rather than "all" but decided the added complexity probably wasn't worth the effort.  Really appreciate you noting my report on the forum and getting this started!